python/dgl/core.py

```diff
@@ -213,7 +213,8 @@ def data_dict_to_list(graph, data_dict, func, target):
         The input graph.
     data_dict : dict[str, Tensor] or dict[(str, str, str), Tensor]]
```

Comment:
add `or Tensor`?

Reply:
and is it better to rename `data_dict` to a more general name?

Reply:
OK, I have added.